Tests/SPMBuildCoreTests/PluginInvocationTests.swift

```diff
@@ -149,10 +150,10 @@ class PluginInvocationTests: XCTestCase {
                         inputFiles: [],
                         outputFiles: [])
                 }
-                
+            */
```

Comment:
left overs?

Reply:
Hmm, I thought I pushed changes to fix this.  I'll double-check.

Reply:
Yes, I definitely fixed this to return actual messages for round-trip testing.  I'll push the latest changes (it wasn't a ton of changes so doesn't invalidate any review, just some last cleanup).